cwl/src/test/scala/wdl4s/cwl/CwlWorkflowWomSpec.scala

```diff
@@ -96,7 +97,52 @@ class CwlWorkflowWomSpec extends FlatSpec with Matchers {
     }
   }
 
-  "A WdlNamespace for 3step" should "provide conversion to WOM" in {
+  behavior of "A WdlNamespace for 3step"
+
+  it should "parse ECMAScript expressions for arguments" in {
+    val wf = decodeAllCwl(rootPath / "three_step.cwl").map {
+      _.select[Workflow].get
+    }.value.unsafeRunSync.fold(error => throw new RuntimeException(s"broken parse! msg was $error"), identity)
+
+    // The second step (aka 1) step should be cgrep
+    val run: WorkflowStep.Run = wf.steps.apply(1).run
+    val commandLineTool: CommandLineTool = run.select[CommandLineTool].get
```

Comment:
ToL: Even in tests, I'd prefer a nice `.getOrElse(fail("..."))` message to a generic `None.get exception`